cwls/cgpRna_with_infuse.cwl

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env cwl-runner
+
+class: Workflow
+
+id: "multi-lane-cgprna-with-infuse-workflow"
+
+label: "A CGP workflow to generate mapping stats, gene counts and fusion events from RNA-seq data using tools in cgpRna"
```

Comment:
You have a `doc` tag later, is label not supposed to be a very short human form of `id`, something like the name of the cwl file?

Not a blocker, just not sure.

Reply:
It's an issue with the online cwl viewer (as I just reported :): https://github.com/common-workflow-language/cwlviewer/issues/228

I can visualize the cwls using my locally deployed cwl viewer.

Reply:
Response attached to wrong comment